SITCOMTN-076.tex

```diff
@@ -49,6 +53,325 @@
+  They should discuss relevant publication policies (e.g., from LSST Science Collaborations) well in advance to avoid possible tensions.
+
+  %\item Since members of the Project team are free to collaborate with others beyond the Project team on science papers that use Early Science data.
+  %Prior to the release of the Early Science data products, the Project team authors are welcome to inform their collaborators beyond the Project team about their intent to write early science papers; transparency is encouraged.
+  %However,
+
+  %scientific results may NOT be shared outside the Project team prior to the associated data release.
+
+  %beyond those directly related to technical and scientific performance
+
+  %\item The authors are welcome to inform their collaborators beyond the Project team about their intent to write early science papers; transparency is encouraged.
+  %However, prior to the public release of data, members of the Project team may not share outside the Project any aspect of any science result obtained with those data.
+  %For example, Project team members may not share with their collaborators outside the project any information about specific science results until the data are public.
+  %\item Members of the Project team are free to collaborate with others beyond the Project team on science papers that use Early Science data.
+  %The authors are welcome to inform their collaborators beyond the Project team about their intent to write such science papers; transparency is encouraged.
+  %Science paper processes, including reviews and approvals, are in the purview of the LSST Science Collaborations.
+
+  \item A technote describing science validation analysis of on-sky commissioning data could become the basis for (parts of) a science paper submitted to a peer-review journal / the arXiv.
+  %Technotes are intended to describe Rubin Observatory infrastructure and science validation work, including analyses of technical and scientific performance during commissioning.
+
+  %\item Scientific results beyond those directly related to technical and scientific performance may NOT be shared outside the Project team prior to the associated data release.
+  %While commissioning data products may NOT be shared prior to release, derived data products may be shared via approved technotes that can then be openly discussed.
+
+  \item The authors are encouraged to circulate a draft within the Project team for comments several weeks in advance of submission / public posting.
+  Since these non-infrastructure publications are not covered by the Project Publication Policy, there is no formal review by Rubin Observatory.
+
+  \item The authors should encourage co-authorship as appropriate to recognize the contributions of other Project team members.
+  No one should be listed as a co-author without their explicit permission.
```

Comment:
Needs to be stronger.   "No one may be listed..."   This may be English v. American.